orca-web/src/main/groovy/com/netflix/spinnaker/orca/controllers/OperationsController.groovy

```diff
@@ -77,15 +77,45 @@ class OperationsController {
 
   @RequestMapping(value = "/orchestrate", method = RequestMethod.POST)
   Map<String, Object> orchestrate(@RequestBody Map pipeline, HttpServletResponse response) {
-    parsePipelineTrigger(executionRepository, buildService, pipeline)
-    Trigger trigger = objectMapper.convertValue(pipeline.trigger, Trigger)
+    if (pipeline.plan) {
+      parseAndValidatePipeline(pipeline)
+      log.info('Not starting pipeline (plan: true): {}', value("pipelineId", pipeline.id))
+      return pipeline
+    }
+
+    Throwable pipelineError = null
```

Comment:
Out of curiosity, why are we dealing with `Throwable` instead of plain ol' `Exception`?

Reply:
Good point; no good reason.  I'll change this to `Exception`